src/main/java/icbm/classic/mods/cubicchunks/CubicChunks.java

```diff
@@ -0,0 +1,60 @@
+package icbm.classic.mods.cubicchunks;
+
+import icbm.classic.ICBMClassic;
+import net.minecraft.world.World;
+import net.minecraftforge.fml.common.Loader;
+
+import java.lang.reflect.Method;
+
+public class CubicChunks {
+
+    public static CubicChunks INSTANCE = new CubicChunks();
+    boolean present = false;
+
+    Method getMinHeightMethod;
+    Method getMaxHeightMethod;
+
+    @SuppressWarnings("JavaReflectionMemberAccess")
+    private CubicChunks() {
+        ICBMClassic.logger().info("ICBMClassic>>CubicChunks: Looking for CubicChunks");
+        if (!Loader.isModLoaded("cubicchunks")) {
+            ICBMClassic.logger().info("ICBMClassic>>CubicChunks: Didn't find CubicChunks");
+            return;
+        }
+
+        ICBMClassic.logger().info("ICBMClassic>>CubicChunks: Found CubicChunks");
+        try {
+            getMinHeightMethod = World.class.getMethod("getMinHeight");
+            getMaxHeightMethod = World.class.getMethod("getMaxHeight");
+            present = true;
+        } catch (Exception e) {
+            ICBMClassic.logger().error("ICBMClassic>>CubicChunks: Failed to hook CubicChunks");
+            e.printStackTrace();
+        }
+    }
+
+    public boolean isPresent() {
+        return present;
+    }
+
+    public int getMinHeight(World world) {
+        try {
+            return (int) getMinHeightMethod.invoke(world);
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
If this fails it will spam chat. Might want to break it out into each exception and silence them after they fail 3-4 times.